### DISCUSSION TITLE
Commandes robots

### DISCUSSION THREAD

**Discussion by zjelt** [robot-soccer-kit/robot-soccer-kit#53] [Q&A] [answered]
Bonjour,
Existe-t-il des méthodes pour commander les leds et le son des robots ?
Jérôme

**Comment by cgaspard3333** [comment, marked as answer]
A l'heure actuelle, il n'existe pas de méthode permettant de commander les LEDs ou les Buzzers des robots.
En effet, nous nous sommes concentrés sur la partie "match" des robots et la commande des LEDs ou Buzzer serait interdite durant ceux-ci (Les LEDs sont utilisées pour identifier les équipes, et les Buzzers pourraient être utilisés pour signifier une faute et/ou pourraient gêner le public en fonction de la programmation faite par les équipes).

Je travaille actuellement à temps complet sur la partie "Arbitrage" afin de la sortir le plus vite possible. 
L'implémentation du contrôle des LEDs et du Buzzer est dans la liste des choses qui suivent pour la portée pédagogique du projet. Cependant ces fonctions seraient, comme je le disais précédemment, désactivées lors d'un match.
De plus, nous devons aussi nous occuper du processus qualité du code pour permettre à des contributeurs externes (à partir de GitHub) de potentiellement apporter de nouvelles fonctionnalités aux robots.

**Reply by zjelt** [reply]
Ok. Effectivement, je te confirme que ma demande est en relation avec une exploitation pédagogique du RSK.
Merci pour ta réponse et bon courage pour la suite.
